.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release Cloudformation
```

Comment:
```suggestion
name: Release
```

Reply:
resolved in [e93dc8f](https://github.com/observeinc/cloudformation-aws-collection/pull/35/commits/e93dc8f8fbfd74a79452f2fa9adadc7dadd41c64)